docs/developers/creating-your-first-plugin.md

```diff
@@ -66,8 +66,8 @@ repositories {
 }
```

Comment:
GH won't let me comment on it since it's right outside of the diff, but the gradle repo example right above this line also needs to be changed.

Reply:
Did not notice that one, should be fixed now.